computerAsk.js

```diff
@@ -1,85 +0,0 @@
-//computer stores number between1-100
```

Comment:
Don't forget to format your code before you push it up

Reply:
roger that thanks GJ